e2e/configs/shared/wdio.sauce.shared.conf.js

```diff
@@ -0,0 +1,29 @@
+const { config } = require('./wdio.shared.conf');
+
+// =================
+// Service Providers
+// =================
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY_EU;
```

Comment:
This should be `process.env.SAUCE_ACCESS_KEY`

Reply:
Done 👍